### PR TITLE
More buttons, more!

### PR DESCRIPTION
**Pull request by specing**
UI needs to be reorganised somehow, its a mess.

### CODE REVIEW

**Comment by specing** [comment]
Is it normal for the llvm part of travis to be broken?

**Comment by specing** [comment]
More unrelated commits.

**Comment by abma** [comment]
https://i.imgflip.com/1wkt5d.jpg

thanks, looks good.no clue why the travis build is currently broken, this change broke it:
https://github.com/springlobby/springlobby/commit/bd18a813f4b96b480b14641cd78e7505e674eb4d#diff-af3b638bc2a3e6c650974192a53c7291L6


i'm trying to fix it, when fixed i'll merge your changes.

File: src/gui/ui.cpp [hunk 1]
```diff
@@ -63,7 +63,7 @@
 
 SLCONFIG("/General/AutoUpdate", true, "Determines if springlobby should check for updates on startup");
 SLCONFIG("/General/LastUpdateCheck", 0L, "Last time springlobby checked for an update");
-SLCONFIG("/GUI/StartTab", (long)MainWindow::PAGE_SINGLEPLAYER, "which tab to show on startup");
+SLCONFIG("/GUI/StartTab", (long)MainWindow::PAGE_BATTLELIST, "which tab to show on startup");
```
Comment (abma) [inline src/gui/ui.cpp]: oh, that was really stupid. thanks, nice catch!

**Comment by abma** [comment]
ah, travis broke it it seems:
https://blog.travis-ci.com/2017-12-12-new-trusty-images-q4-launch

**Comment by abma** [comment]
thanks for the changes!

**Pull request merged**